utils.c: In function 'delay_usec':
utils.c:108:20: error: storage size of 'tv' isn't known
     struct timeval tv;
                    ^~
diff --git a/src/utils.c b/src/utils.c
index 1dac15c..3e2fa6a 100644
--- src/utils.c
+++ src/utils.c
@@ -21,6 +21,7 @@
 #include "sysdeps.h"
 #include "utils.h"
 #include <time.h>
+#include <sys/time.h>
 #include <errno.h>
 
 #define DEBUG 1

